docs/source/getting_started/platform.rst

```diff
@@ -264,42 +264,42 @@ you can use a simple script to translate it to the coco format:
 
 
 
-Sampling
-----------------
+Selection
+---------
 
-Before you start sampling make sure you have created a dataset 
+Before you start selecting make sure you have created a dataset
 and uploaded images and embeddings. See `Create a Dataset`_.
 
-Now, let's get started with sampling!
+Now, let's get started with selecting!
 
-Follow these steps to sample the most representative images from your dataset:
+Follow these steps to select the most representative images from your dataset:
 
 #. Choose the dataset you want to work on from the *"My Datasets"* 
 section by clicking on it.
 
 #. Navigate to *"Analyze & Filter"* → *"Sampling"* through the menu on the left.
 
-#. Choose the embedding and sampling strategy to use for this sampling run.
+#. Choose the embedding and selection strategy to use for this selection.
 
-#. Give a name to your subsampling so that you can later compare 
-   the different samplings.
+#. Give a name to your selectiong so that you can later compare
+   the different selections.
 
-#. Hit "Process" to start sampling the data. Each sample is now assigned an 
-   "importance score". The exact meaning of the score depends on the sampler.
+#. Hit "Process" to start selecting the data. Each sample is now assigned an
+   "importance score". The exact meaning of the score depends on the selection strategy.
```

Comment:
Maybe we can leave " The exact meaning of the score depends on the selection strategy." away